src/modules/menu/components/activeLayers/MenuActiveLayersListItemTimeSelector.vue

```diff
@@ -2,6 +2,7 @@
     <PopoverButton
         v-if="hasMultipleTimestamps"
         ref="popover"
+        popover-position="left"
```

Comment:
Why do you want the popover to open on the left?

Reply:
that's to fix an issue with timestamp testing in Cypress, if set to `auto` (left untouched) it would go on the top position, and with the dev warning, Cypress considers that it is covered by the disclaimer and fails the test of visibility. That was an "easy" fix.
We could move it to the right though...